integration-tests/tests/fixtures/integration_test_config.py

```
@@ -0,0 +1,28 @@
+def integration_test_config() -> IntegrationTestConfig:
+    """Fixture that provides an IntegrationTestConfig shared across tests."""
+    core_config = CoreConfig(
+        clp_core_bins_dir=Path(get_env_var("CLP_CORE_BINS_DIR")).expanduser().resolve()
+    )
+    package_config = PackageConfig(
+        clp_package_dir=Path(get_env_var("CLP_PACKAGE_DIR")).expanduser().resolve()
+    )
+    return IntegrationTestConfig(
+        core_config=core_config,
+        package_config=package_config,
+        test_root_dir=Path(get_env_var("CLP_BUILD_DIR")).expanduser().resolve() / "integration-tests",
```

Ruff (0.12.2)

Line too long (102 > 100)

(E501)